assets/js/googlesitekit/datastore/user/feature-tours.js

```diff
@@ -153,6 +189,13 @@ const baseControls = {
 
 		return true;
 	} ),
+	[ CACHE_LAST_DISMISSED_AT ]: createRegistryControl( ( ) => async ( { payload } ) => {
```

Comment:
This doesn't need to be a registry control since it doesn't use the registry, it can simply be the inner `async` function.